quic/s2n-quic-core/src/recovery/bbr/round.rs

```diff
@@ -0,0 +1,155 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::recovery::bandwidth::PacketInfo;
+
+//= https://tools.ietf.org/id/draft-cardwell-iccrg-bbr-congestion-control-02#4.5.1
+//# Several aspects of the BBR algorithm depend on counting the progress of "packet-timed" round
+//# trips, which start at the transmission of some segment, and then end at the acknowledgement
+//# of that segment. BBR.round_count is a count of the number of these "packet-timed" round trips
+//# elapsed so far.
+#[derive(Clone, Debug, Default)]
+pub(crate) struct Counter {
+    /// The `delivered_bytes` at which the next round begins
+    next_round_delivered_bytes: u64,
+    /// True if the current ack being processed started a new round
+    round_start: bool,
+    /// The number of rounds counted since initialization
+    round_count: u64,
+}
+#[allow(dead_code)] // TODO: Remove when used
```

Comment:
can you move this to `fn round_count` specifically or just remove that fn and add it back if you need it.